tests/providers/microsoft/azure/hooks/test_azure_container_instance.py

```diff
@@ -110,3 +121,17 @@ def test_connection_failure(self, mock_container_groups_list):
         status, msg = self.hook.test_connection()
         assert status is False
         assert msg == "Authentication failed."
+
+    @patch("azure.common.credentials.ServicePrincipalCredentials")
+    @patch("airflow.providers.microsoft.azure.hooks.container_instance.DefaultAzureCredential")
+    def test_get_conn_fallback_to_default_azure_credential(
+        self,
+        mock_default_azure_credential,
+        mock_service_pricipal_credential,
+        connection_without_login_password_tenant_id,
+    ):
+        hook = AzureContainerInstanceHook(azure_conn_id=connection_without_login_password_tenant_id.conn_id)
+        hook.get_conn()
```

Comment:
Should the return value be validated?

Reply:
Sure. I just check the return value is mocked client